packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx

```diff
@@ -1,5 +1,4 @@
 import {MenuItem, TextField, makeStyles} from "@material-ui/core"
-import {getLanguages} from "../../lib/contentful-api"
```

Comment:
Why is this removed?

Reply:
Ah, in my branch it was unused. Good shout, adding back in.